package-lock.json

```diff
@@ -847,7 +856,7 @@
             "dependencies": {
                 "chalk": {
                     "version": "1.1.3",
-                    "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+                    "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
why this change?

Reply:
I wanted to observe something and hence pushed this auto generated change.
This change got generated when built on MAC. I'll revert this as on windows it is updating it back to https.